index.html

```diff
@@ -37,36 +37,31 @@
             var c = document.getElementById("textcanvas");
             var cc = c.getContext('2d');
             var length;
+            
             function random(min,max) {
                 return Math.floor(Math.random() * (max - min) + min);
             }
+
+            String.prototype.removeAt = function (index) {
+                return this.substr(0, index) + this.substr(index + 1, this.length - index);
+            }
+
             function scramble(word) {
-                var charToAdd;
-                length = realword.length;
-               for(var i = 0; i < length;) {
-                     charToAdd = word.charAt(random(0,length));
-                    if(scrambled.includes(charToAdd) || scrambled == realword) {
-                        i = length;
-                        scrambled = [];
-                        scramble(realword);
-                    } else {
-                        scrambled[i] = charToAdd;
-                        i = i + 1;
-                    }
-                    
-               }
+                var scrambled = "";
+                while (word.length > 0) {
+                    var index = random(0, word.length);
+                    var char = word.charAt(index);
+                    scrambled += char;
+                    word = word.removeAt(index);
+                }
+                return scrambled;
             }
+    
             let wordlist = ["break","clean","clash","watch","punch","water","computer","flashed","grumpy","weird","lighten","fruit","drink","group","stink","world","about","cats","dogs","trucks","monkey",
-            "jumble","boat","bugs","bounce","wrote","market","push","spear","orange","block","tired","run","double","wars","mile","pencil","dough","number","cube"];
-            let scrambled = [];
-            var realword = new String(wordlist[random(0,39)]);
-            scramble(realword);
-            var scrambledText = "";
-            for(var j = 0; j < length;) {
-                scrambledText += scrambled[j]; 
-                j=j+1;
-            }
-            
+            "jumble","boat","bugs","bounce","wrote","market","push","spear","orange","block","tired","run","double","wars","mile","pencil","dough","number","cube","popsicle"];
+
+            var realword = new String(wordlist[random(0,wordlist.length)]);

```

Comment:
I updated this line to use `wordlist.length` as the max value passed to `random`. This way, this line doesn't need to be updated again when adding new words in the future.